terraform/gitops/generate-files/templates/pm4ml/rbac.yaml.tpl

```diff
@@ -26,4 +26,16 @@ spec:
           "subject_id": "{{ print .Subject }}"
         }
   mutators:
-    - handler: header
\ No newline at end of file
+    - handler: header
+---
+apiVersion: mojaloop.io/v1
+kind: MojaloopRole
+metadata:
+  name: pm4mladmin
+  namespace: ${ory_namespace}
+spec:
+  role: pm4mladmin
+  permissions:  ## Todo: decide which permissions the role should have
+    - pm4mlViewTransfers
+    - pm4mlApi
+---
```

Comment:
We can move this to ory folder and define a json file similar to mojaloop for maintaining roles permission mappings.

Reply:
the file will be similar to this one ./terraform/k8s/mojaloop-rbac-permissions.yaml

Reply:
done